deploy.sh

```diff
@@ -176,9 +176,18 @@ build_app () {
 }
 
 build_crypto_bench () {
+  # Flatten the array
+  # This is equivalent to the following python snippet: ' '.join(arr).replace(' ', ',')
+  local feature_list=$(IFS=$'\n'; echo "$@")
+  if [ "X${feature_list}" != "X" ]
+  then
+    feature_list="${feature_list// /,}"
+  fi
```

Comment:
I believe this code should not be duplicated.

```sh
comma_separated () {
  local feature_list=$(IFS=$'\n'; echo "$@")
￼  if [ "X${feature_list}" != "X" ]
￼  then
￼    echo "${feature_list// /,}"
￼  fi
}
```

We would then have `local feature_list="$(comma_separated "$@")"` in both `build_app` and here.

Reply:
I had to tweak that a bit but indeed refactored to avoid code duplication.